### PR TITLE
fixed audioclipplayer bug

### PR DESCRIPTION
**Pull request by Tzyinc**

### CODE REVIEW

File: MIDIchlorians/AudioClipPlayer.swift [hunk 1]
```diff
@@ -18,9 +18,9 @@ struct AudioClipPlayer {
         guard let soundURL = Bundle.main.url(forResource: audioDir, withExtension: ext) else {
             return nil
         }
-        let sysID: SystemSoundID = 0
-        var testSound: SystemSoundID = 0
-        AudioServicesCreateSystemSoundID(soundURL as CFURL, &testSound)
+
+        var sysID: SystemSoundID = 0
+        AudioServicesCreateSystemSoundID(soundURL as CFURL, &sysID)
```
Comment (ngzhian) [inline MIDIchlorians/AudioClipPlayer.swift:23]: yes this is good! once this is merged then we can have sounds :)

File: MIDIchlorians/MIDIchlorians.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -158,14 +159,20 @@
...
 		33EC0A431E6FE67700229A92 /* AnimationSequence.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = AnimationSequence.swift; sourceTree = "<group>"; };
-		68381907BD5B384221D4EB3C /* Pods-MIDIchlorians.debug.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-MIDIchlorians.debug.xcconfig"; path = "Pods/Target Support Files/Pods-MIDIchlorians/Pods-MIDIchlorians.debug.xcconfig"; sourceTree = "<group>"; };
+		3B2A96A7E2F93F5485F686BB /* Pods-MIDIchloriansUITests.release.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-MIDIchloriansUITests.release.xcconfig"; path = "Pods/Target Support Files/Pods-MIDIchloriansUITests/Pods-MIDIchloriansUITests.release.xcconfig"; sourceTree = "<group>"; };
 		805946D11E816D0E00EF7A2B /* AudioClipPlayer.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = AudioClipPlayer.swift; sourceTree = "<group>"; };
+<<<<<<< HEAD
```
Comment (ngzhian) [inline MIDIchlorians/MIDIchlorians.xcodeproj/project.pbxproj]: uh there's a merge conflict here, need to delete this

File: MIDIchlorians/MIDIchlorians.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -158,14 +159,20 @@
...
 		CD9072948CB8AB76A19799BB /* Pods_MIDIchloriansUITests.framework */ = {isa = PBXFileReference; explicitFileType = wrapper.framework; includeInIndex = 0; path = Pods_MIDIchloriansUITests.framework; sourceTree = BUILT_PRODUCTS_DIR; };
 		E6848E6E722EA2AD451EA962 /* Pods-MIDIchloriansUITests.release.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-MIDIchloriansUITests.release.xcconfig"; path = "Pods/Target Support Files/Pods-MIDIchloriansUITests/Pods-MIDIchloriansUITests.release.xcconfig"; sourceTree = "<group>"; };
 		E9158EAA1E860CDB00B7ADF9 /* GridController.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = GridController.swift; sourceTree = "<group>"; };
 		E9158EAF1E86353600B7ADF9 /* GridCollectionViewController.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = GridCollectionViewController.swift; sourceTree = "<group>"; };
 		E9158EB11E8645F800B7ADF9 /* PadView.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = PadView.swift; sourceTree = "<group>"; };
+>>>>>>> master
```
Comment (ngzhian) [inline MIDIchlorians/MIDIchlorians.xcodeproj/project.pbxproj]: same here, merge conflict artefacts

File: MIDIchlorians/MIDIchlorians.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -361,6 +356,20 @@
 			path = ..;
 			sourceTree = "<group>";
 		};
+<<<<<<< HEAD
```
Comment (ngzhian) [inline MIDIchlorians/MIDIchlorians.xcodeproj/project.pbxproj]: in this file as well

File: MIDIchlorians/MIDIchlorians.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -639,53 +649,53 @@
...
 			inputPaths = (
 			);
-			name = "[CP] Embed Pods Frameworks";
+			name = "[CP] Check Pods Manifest.lock";
 			outputPaths = (
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-MIDIchloriansUITests/Pods-MIDIchloriansUITests-frameworks.sh\"\n";
+			shellScript = "diff \"${PODS_ROOT}/../Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n";
```
Comment (ngzhian) [inline MIDIchlorians/MIDIchlorians.xcodeproj/project.pbxproj:729]: i think you have an error in your pod file, make sure you open the xcworkspace, and also pod install

**Review by ngzhian — APPROVED** [review]

**Comment by ngzhian** [comment]
LGTM!

**Comment by Tzyinc** [comment]
@advaypal added some lines into session to load the audio from config
@ngzhian updated two lines in GridController since AudioManager is now a singleton

**Pull request merged**